### PR TITLE
Pt/quote

### PR DESCRIPTION
**Pull request by paaaatrick**
## Trello Card(s)
https://trello.com/c/9Cs3RnXR/25-section-quote-callout

## What was updated?
Quote section template and styles added.

## Screenshots of updates (visual changes)

Desktop
Screenshot 2019-11-27 09 02 52

Mobile
Screenshot 2019-11-27 09 03 13

### CODE REVIEW

File: web/themes/fire/templates/components/quote/_quote.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  @include media-breakpoint-down(sm) {
+    @include make-col(12);
+  }
+
+  &__wrapper {
+    @include make-col(8);
+    padding: 25px 25px 10px;
+    margin: 0 auto;
+    text-align: center;
+    border: 1px solid #7c878e;
```
Comment (ryanschmidt) [inline web/themes/fire/templates/components/quote/_quote.scss]: We should be using variables for all colors cc @theroadeldorado

File: web/themes/fire/templates/components/quote/paragraph--quote.html.twig [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    {% endif %}
+    {% if author %}
+      <h3 class="quote__author">{{ author }}</h3>
+    {% endif %}
+    {% if author_title %}
+      <h5 class="quote__author-title" >{{ author_title }}</h5>
+    {% endif %}
+  </div>
+</div>
+
```
Comment (ryanschmidt) [inline web/themes/fire/templates/components/quote/paragraph--quote.html.twig]: What's up with the extra lines?

File: web/themes/fire/templates/components/quote/_quote.scss (hunk 1 above)
Comment (paaaatrick) [reply web/themes/fire/templates/components/quote/_quote.scss]: Screenshot 2019-11-27 10 59 04

Changed each to use theme-color()

File: web/themes/fire/templates/components/quote/paragraph--quote.html.twig (hunk 2 above)
Comment (paaaatrick) [reply web/themes/fire/templates/components/quote/paragraph--quote.html.twig]: they need to go, removed them.

**Pull request merged**